robot_ws/.rosinstall

```diff
@@ -1,9 +1,6 @@
 # Amazon ROS HelloWorld package dependencies
 - git: {local-name: src/deps/turtlebot3, uri: 'https://github.com/ROBOTIS-GIT/turtlebot3.git', version: ros2}
 - git: {local-name: src/deps/turtlebot3_msgs, uri: 'https://github.com/ROBOTIS-GIT/turtlebot3_msgs.git', version: ros2}
-- git: {local-name: src/deps/cartographer/cartographer, uri: 'https://github.com/ROBOTIS-GIT/cartographer.git', version: dashing}
-- git: {local-name: src/deps/cartographer/pcl_conversions, uri: 'https://github.com/ros2/pcl_conversions.git', version: ros2}
 - git: {local-name: src/deps/cartographer/cartographer_ros, uri: 'https://github.com/ROBOTIS-GIT/cartographer_ros.git', version: dashing}
```

Comment:
Why did this cartographer dependency survive?

Reply:
Unsure. They kept it in https://github.com/ROBOTIS-GIT/turtlebot3/pull/472/files#diff-2b45d515563d9cf3dba9fdb53eb1e5efR14 so I kept it the same.

Reply:
fair enough